### PR TITLE
[Tensor] rename some APIs in TensorSpec and Polish view unittest

### PR DESCRIPTION
**Pull request by feifeibear**

### CODE REVIEW

File: colossalai/nn/_ops/embedding.py [hunk 1]
```diff
@@ -32,6 +32,8 @@ def colo_embedding_1Dcol(input_tensor: ColoTensor,
     output = ColoTensor.from_torch_tensor(output_parallel, spec=output_spec)
 
     compute_spec = weight.tensor_spec.compute_spec
+
+    return output
```
Comment (ver217) [inline colossalai/nn/_ops/embedding.py:36]: Why ignore `compute_spec.output_replicate`?

Comment (Wesley-Jzy) [inline colossalai/nn/_ops/embedding.py:36]: Why return here

Comment (feifeibear) [reply colossalai/nn/_ops/embedding.py:36]: I added it back

File: colossalai/tensor/chunk.py [hunk 2]
```diff
@@ -228,6 +230,7 @@ def copy_tensor_to_chunk_slice(self, tensor: torch.Tensor, data_slice: torch.Ten
             data_slice (torch.Tensor): the tensor to be copied to the chunk
         """
         tensor_info = self.tensors_info[tensor]
+        assert type(data_slice) == torch.Tensor, "copy_tensor_to_chunk_slice must use a torch tensor"
```
Comment (ver217) [inline colossalai/tensor/chunk.py:233]: I think `ColoTensor` is also allowed.

File: colossalai/nn/_ops/embedding.py (hunk 1 above)
Comment (feifeibear) [reply colossalai/nn/_ops/embedding.py]: reverted

File: colossalai/tensor/chunk.py (hunk 2 above)
Comment (feifeibear) [reply colossalai/tensor/chunk.py]: DONE

**Review by zxgx — APPROVED** [review]

**Review by Wesley-Jzy — APPROVED** [review]

**Review by ver217 — APPROVED** [review]

**Comment by feifeibear** [comment]
image

**Pull request merged**